Document temporality presets in Python (#2551)
open-telemetry/opentelemetry.io: content/en/docs/instrumentation/python/exporters.md

each instrument kind, or change the default aggregation for each instrument
kind.

#### Temporality Presets

There are temporality presets for each instrumentation kind. These presets can
be set with the environment variable
`OTEL_EXPORTER_METRICS_TEMPORALITY_PREFERENCE`, for example:

> ```console
> $ export OTEL_EXPORTER_METRICS_TEMPORALITY_PREFERENCE="DELTA"

Reviewer: ```suggestion
```sh
export OTEL_EXPORTER_METRICS_TEMPORALITY_PREFERENCE="DELTA"
```

(Note: you don't actually need the quotes in the above statement, but it's ok if you want to keep them.)

```

The default value for `OTEL_EXPORTER_METRICS_TEMPORALITY_PREFERENCE` is

> `"CUMULATIVE"`.

Reviewer: ```suggestion
`CUMULATIVE`.
```

The available values and their corresponding settings for this environment
variable are:

- `CUMULATIVE`

  - `Counter`: `CUMULATIVE`
  - `UpDownCounter`: `CUMULATIVE`
  - `Histogram`: `CUMULATIVE`
  - `ObservableCounter`: `CUMULATIVE`
  - `ObservableUpDownCounter`: `CUMULATIVE`
  - `ObservableGauge`: `CUMULATIVE`

- `DELTA`

  - `Counter`: `DELTA`
  - `UpDownCounter`: `CUMULATIVE`
  - `Histogram`: `DELTA`
  - `ObservableCounter`: `DELTA`
  - `ObservableUpDownCounter`: `CUMULATIVE`
  - `ObservableGauge`: `CUMULATIVE`

- `LOWMEMORY`
  - `Counter`: `DELTA`
  - `UpDownCounter`: `CUMULATIVE`
  - `Histogram`: `DELTA`
  - `ObservableCounter`: `CUMULATIVE`
  - `ObservableUpDownCounter`: `CUMULATIVE`
  - `ObservableGauge`: `CUMULATIVE`

> Setting `OTEL_EXPORTER_METRICS_TEMPORALITY_PREFERENCE` to any other value than
> `CUMULATIVE`, `DELTA` or `LOWMEMORY` will log a warning and set this environment
> variable to `CUMULATIVE`.

Reviewer: ```suggestion
If you set `OTEL_EXPORTER_METRICS_TEMPORALITY_PREFERENCE` to a value other than
`CUMULATIVE`, `DELTA`, or `LOWMEMORY`, then a warning will be logged and the variable
value will fall back to the default of `CUMULATIVE`.
```

You might want to say how/what is logging the warning?
